packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -263,6 +196,26 @@ export class VirtualNode extends VirtualNodeBase {
       resourceName: this.physicalName,
     });
   }
+
+  /**
+   * Utility method to add an inbound listener for this VirtualNode
+   */
+  public addListener(...listeners: VirtualNodeListener[]) {
+    listeners.forEach(listener => this.listeners.push(listener.bind(this)));
+  }
+
+  /**
+   * Add a Virtual Services that this node is expected to send outbound traffic to
+   */
+  public addBackends(...props: IVirtualService[]) {
```

Comment:
Let's make `addBackends()` consistent with `addListener()` (call it `addBackend()`, make it take a single `IVirtualService`).

We will need a breaking change in the PR description for this too.